pkg/config/config.go

```diff
@@ -1040,6 +1040,7 @@ type Docker struct {
 	BuildFlagTemplates []string `yaml:"build_flag_templates,omitempty" json:"build_flag_templates,omitempty"`
 	PushFlags          []string `yaml:"push_flags,omitempty" json:"push_flags,omitempty"`
 	Use                string   `yaml:"use,omitempty" json:"use,omitempty" jsonschema:"enum=docker,enum=buildx,default=docker"`
+	DepotProject       string   `yaml:"depot_project,omitempty" json:"depot_project,omitempty"`
```

Comment:
Assuming we might add more things later (either for depot or for something else), maybe something like:

```yaml
dockers:
- use: depot
  depot:
    project: foo
```

wdyt?

Reply:
@caarlos0 Sounds great !  I've added that in [27eba41](https://github.com/goreleaser/goreleaser/pull/4397/commits/27eba41e969db2c8cd3f0a3112af7a52bfe7261a)